scss/_base_code.scss

```diff
@@ -0,0 +1,28 @@
+// Base code styles
+@mixin vf-b-code {
+  code,
+  samp,
+  kbd {
+    font-family: $font-monospace;
+    text-align: left;
+  }
+
+  pre,
+  code {
+    text-align: left;
+    white-space: pre-line;
+    word-spacing: normal;
+    word-wrap: break-word;
+    tab-size: 4;
+    hyphens: none;
+    direction: ltr;
+  }
+
+  pre {
+    background: $color-dark;
+    color: $color-light;
```

Comment:
Yes - see spec in the ticket: https://github.com/ubuntudesign/vanilla-framework/issues/398